.eslintrc.js

```diff
@@ -23,6 +23,7 @@ module.exports = {
     'class-methods-use-this': 'off',
     'no-underscore-dangle': 'off',
     '@typescript-eslint/type-annotation-spacing': ['error'],
-    'linebreak-style': ["error", "windows"]
```

Comment:
Une technique pour éviter les git diff quand tu rajoute un élément dans une liste / objet et de mettre une virgule au dernier élement. 
cf https://eslint.org/docs/rules/comma-dangle

> Another argument in favor of trailing commas is that it improves the clarity of diffs when an item is added or removed from an object or array:

(C'est juste pour l'info, perso je m'en fous)

Reply:
Aight, I'll do it from now on.